go/types/blob.go

```diff
@@ -172,10 +186,46 @@ func chunkBlobLeaf(vr ValueReader, buff []byte) (Collection, orderedKey, uint64)
 	return blob, orderedKeyFromInt(len(buff)), uint64(len(buff))
 }
 
+// NewBlob creates a Blob by reading from r.
 func NewBlob(r io.Reader) Blob {
 	return NewStreamingBlob(r, nil)
 }
 
+// NewBlobP creates a Blob by reading from every Reader in rs in parallel, then
```

Comment:
I'm considering just making NewBlob/NewStreamingBlob do the parallelisation automatically, and not having NewBlobP. WDYT?